### PR TITLE
TASK: Neos 5.0 compatibility

### PR DESCRIPTION
**Pull request by sbruggmann**

### CODE REVIEW

**Comment by paavo** [comment]
@dfeyer any plans to merge this PR?
i'd love to use DimensionKeeper with Neos 5

**Comment by paavo** [comment]
Maybe @dimaip or @rolandschuetz can merge @sbruggmann PR?

**Comment by rolandschuetz** [comment]
I'm not part of treeagency...

**Comment by paavo** [comment]
> I'm not part of treeagency...

I know 😆 
But i saw you already commited to 'ttreeagency/DimensionKeeper' so i thought you maybe have access to the repository. 

I can't reach @dfeyer to accept the PR.

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "type": "neos-package",
     "license": "MIT",
     "require": {
-        "neos/content-repository": "^3.0 || ^4.0"
+        "neos/content-repository": "^3.0 || ^4.0 || ^5.0"
```
Comment (dfeyer) [inline composer.json]: Did you check if `PsrSystemLoggerInterface` is available in 3, 4 ?

**Comment by dfeyer** [comment]
Sorry for the delay ... are you sure the PsrSystemLoggerInterface is available in old version like 3.0 and 4.0 ? Except that, I'm fine with this PR

**Comment by paavo** [comment]
> Sorry for the delay ... are you sure the PsrSystemLoggerInterface is available in old version like 3.0 and 4.0 ? Except that, I'm fine with this PR

Thanks for having a look at this 👍 
Let me ask @sbruggmann tonight (Neos Meetup at JVM if you want to join @dfeyer )

**Comment by dfeyer** [comment]
>  Let me ask @sbruggmann tonight (Neos Meetup at JVM if you want to join @dfeyer )

@paavo  Unfortunatly I can't, be say hello to all the neos people

**Comment by paavo** [comment]
> @paavo Unfortunatly I can't, be say hello to all the neos people

I just saw its an [online Meetup](https://www.meetup.com/Neos-CMS-and-Flow-Framework-Meetup-Swiss/events/kssdmrybcmbwb/) if the location is the issue.

File: composer.json (hunk 1 above)
Comment (sbruggmann) [reply composer.json]: Ah you're right of course.
I've removed 3.0 now from the dependencies.

The PsrSystemLoggerInterface was introduced in flow 5.0, so CR 4.0 is fine.

**Comment by sbruggmann** [comment]
@dfeyer @paavo Thanks for taking care of this.

The PsrSystemLoggerInterface version issue is solved now.

I think, it will be okay to not be backward compatible with the latest package version.

**Comment by paavo** [comment]
good morning @dfeyer 
looks like @sbruggmann PR is ready to be merged ☝️ 

> The PsrSystemLoggerInterface version issue is solved now.

**Pull request merged**